src/Shared/Recruit.Vacancies.Client/Application/Communications/CompositeDataItemProviderPlugins/ApplicationsSubmittedCompositeDataItemPlugin.cs

```diff
@@ -57,7 +57,7 @@ private string GetNoApplicationsSubmittedHeader(IEnumerable<CommunicationMessage
             var totalNoOfVacancies = vacancyRefs.Count();
             var totalNoOfApplications = messages.Count();
 
-            var header = $"There has been {totalNoOfApplications} new {"application".ToQuantity(totalNoOfApplications, ShowQuantityAs.None)} for {totalNoOfVacancies} {"vacancy".ToQuantity(totalNoOfVacancies, ShowQuantityAs.None)}";
+            var header = $"There has been {totalNoOfApplications} new {"application".ToQuantity(totalNoOfApplications, ShowQuantityAs.None)} for {totalNoOfVacancies} {"advert".ToQuantity(totalNoOfVacancies, ShowQuantityAs.None)}";
             return header;
```

Comment:
based on the AC's should this be "apprenticeship adverts" rather than just "adverts"?

Reply:
No AC -> in the aggregate header: There has been [xx] new applications for [xx] adverts

Reply:
agree, I mis-read